client/asset/eth/eth.go

```diff
@@ -711,14 +726,14 @@ func (eth *ExchangeWallet) Swap(swaps *asset.Swaps) ([]asset.Receipt, asset.Coin
 		totalContractValue += contract.Value
 	}
 
-	fees := dexeth.InitGas(1, swaps.AssetVersion) * uint64(len(swaps.Contracts)) * swaps.FeeRate
+	fees := dexeth.InitGas(1, swaps.AssetVersion) * uint64(len(swaps.Contracts)) * gasFeeLimit
 
 	totalSpend := fees + totalContractValue
 	if totalSpend > totalInputValue {
-		return nil, nil, 0, fmt.Errorf("unfunded contract. %d < %d", totalInputValue, totalSpend)
+		return fail(fmt.Errorf("unfunded contract. %d < %d", totalInputValue, totalSpend))
 	}
 
-	tx, err := eth.node.initiate(eth.ctx, swaps.Contracts, swaps.FeeRate, swaps.AssetVersion)
+	tx, err := eth.node.initiate(eth.ctx, swaps.Contracts, gasFeeLimit, swaps.AssetVersion)
```

Comment:
Can't do this part. This is validated by the server.

Reply:
More than that, the "fee limit" was only ever intended to **block order funding**, nothing else.  In fact, this is the setting that I fought to keep out of the wallet since Core's trade methods (prepareTrackedTrade) could have just as easily applied this user set limit in the one place it was intended.  https://github.com/decred/dcrdex/pull/874#discussion_r545420396